tests/drivers/ipm/src/main.c

```diff
@@ -105,5 +106,6 @@ void main(void)
 
 	rv = TC_PASS;
 	TC_END_RESULT(rv);
+	TC_SUITE_END("test_ipm", (rv != TC_PASS));
```

Comment:
Coverity is probably going to complain about the second argument as the macro expects a numeric value.

Reply:
> Coverity is probably going to complain about the second argument as the macro expects a numeric value.

Ok, reworked to be more type-safe and consistent with existing result-taking macros.